### PR TITLE
Release Candidate v67.1.0

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,125 @@
+# 67.1.0
+
+In this minor release we provide a shadow opacity reset toggle for Flexible Header, VoiceOver and rounded corners support for the Navigation Drawer, and lastly a bug fix regarding MDCShadowLayer with Corner Radius along with other bug fixes and unit tests improvements.
```
Comment (romoore) [inline CHANGELOG.md]: Tip: Looks like this line is really long? Should we limit to 80 chars?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+
+## New features
+
+### Flexible Header
+* Now with Flexible Header you can enables resetting the visible shadow opacity to 0 when setting trackingScrollView to 0 by setting the BOOL property `resetShadowAfterTrackingScrollViewIsReset`.
```
Comment (romoore) [inline CHANGELOG.md]: Is this correct, "setting trackingScrollView to 0"? Should this be "0" be "`nil`"?

Comment (romoore) [inline CHANGELOG.md]:
```suggestion
* Flexible Header supports resetting the visible shadow opacity to 0...
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,125 @@
...
+
+### Navigation Drawer
+* When VoiceOver or Switch Control is turned on in your app, Navigation Drawer will now always show up in full screen.
```
Comment (romoore) [inline CHANGELOG.md]:
```suggestion
* When VoiceOver or Switch Control is turned on in your app, Navigation Drawer will always show up in full screen.
```

Comment (romoore) [inline CHANGELOG.md]: Is this correct? We make the drawer full screen even if the VC is only 100 points high?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,125 @@
...
+
+* Navigation Drawer now has rounded corners when there is more scrollable content to scroll in the drawer.
```
Comment (romoore) [inline CHANGELOG.md]:
```suggestion
* Navigation Drawer has rounded corners when there is more scrollable content to scroll in the drawer.
```

**Review by romoore — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (yarneo) [reply CHANGELOG.md]: yes that's correct.

**Pull request merged**